wrangler/src/fees/get.js

```diff
@@ -0,0 +1,30 @@
+import { response } from 'cfw-easy-utils'
+import { authTxToken } from '../@utils/auth'
+
+export default async ({ request, params, env }) => {
+  const { TX_FEES, STELLAR_NETWORK } = env
+  const { publicKey } = params
+
+  const feeToken = request.headers.get('authorization')?.split(' ')?.[1]
+
+  const { publicKey: authedPublicKey } = authTxToken(STELLAR_NETWORK, feeToken)
+
+  if (authedPublicKey !== publicKey) {
```

Comment:
If we decide to remove the URL param, I suggest we change the URL to `GET /fee` or `GET /feeBalance` in this case. As a user, I would be confused leaving it as `GET /fees` , since it no longer really defines a group of things.

I still sorta like having strongly defined resources by URL params, where the auth exists only to define access. But, I don't feel too strongly about this.

Reply:
I do feel pretty strongly that it should _not_ be included, which I believe is the case now? Unnecessary redundancy only wastes resources and leads to confusion. I say scrap it and if there's push back or ever a good reason to revert we do it then not now.